packages/react/src/primitives/types/style.ts

```diff
@@ -16,7 +17,7 @@ export type ResponsiveStyle<PropertyType> =
   | PropertyType[]
   | ResponsiveObject<PropertyType>;
 
-export interface BaseStyleProps {
+export interface BaseStyleProps extends GridItemStyleProps {
```

Comment:
This allows all primitives to be used as Grid items

Reply:
great!